lib/breathe_client.rb

```diff
@@ -42,11 +68,79 @@ def events(after:)
         hash[email] = EventCollection.new(events)
       }
     end
-    memo_wise :events
+    memo_wise :absence_events
+
+    def sickness_events(after:)
```

Comment:
It feels like we've got a some repetition now between the different event types – the `x_by_email` method for example. Is there some refactoring that could be done to make it a bit DRYer?

Reply:
Probably, but I don't have a good way to call out the API difference between sickness / training and absence. Do you?